### PR TITLE
Create customer fixed.

### PR DESCRIPTION
**Pull request by elvixpix**

### CODE REVIEW

File: api/domain/customersDomain.js [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
-        throw new Error('Provide Customer object to create new Customer.');
+    let newCustomer;
+
+    if (!customerData.name || !/\S/.test(customerData.name)) {
+        throw new Error('Name can\'t be empty');
     }
 
     try {
-        return await customerRepo.createCustomer(customerData);
+        newCustomer = await customerRepo.createCustomer(customerData);
```
Comment (bionanek) [inline api/domain/customersDomain.js]: hmmm what's the reason you assign it here and return after `try...catch` instead of returning it inside try?

Comment (elvixpix) [reply api/domain/customersDomain.js]: Oh yea, I see now. Ofc

Comment (elvixpix) [reply api/domain/customersDomain.js]: done

Comment (elvixpix) [reply api/domain/customersDomain.js]: NOT YET

Comment (elvixpix) [reply api/domain/customersDomain.js]: Now

**Pull request merged**